storage/dataflux/worksteal.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dataflux
+
+import (
+	"context"
+	"sync"
+
+	"cloud.google.com/go/storage"
+)
+
+// WorkerStatus indicates the status of a worker.
+type WorkerStatus int
```

Comment:
Missed this earlier -- can you make both this and the const values unexported? It looks like they are not in any public surfaces currently.